sphinx/source/docs/dev_guide/documentation.rst

```diff
@@ -67,6 +70,27 @@ inserting an implicit "IS" makes a complete sentence. For example::
 
 can be reasonably read as "title_font IS a font used for the plot title".
 
+Headings in pages
+~~~~~~~~~~~~~~~~~
+
+When working on narrative documentation, you may use headings to split up your
+work. Headings are used in the following way
+
+.. code-block:: sh
+  
+    Top level (makes a top-level entry in the side-bar)
+    ====================================================
```

Comment:
being pedantic but only because Sphinx would also complain: there is one extra `=` too many